HerPublicWebsite/Startup.cs

```diff
@@ -175,6 +178,11 @@ private void ConfigureGovUkNotify(IServiceCollection services)
             services.AddScoped<IEmailSender, GovUkNotifyApi>();
             services.Configure<GovUkNotifyConfiguration>(
                 configuration.GetSection(GovUkNotifyConfiguration.ConfigSection));
+            services.AddScoped<INotificationClient, NotificationClient>(serviceProvider =>
+            {
+                var govUkNotifyConfiguration = serviceProvider.GetService<IOptions<GovUkNotifyConfiguration>>();
+                return new NotificationClient(govUkNotifyConfiguration.Value.ApiKey);

```

Comment:
Have you checked that this configuration works as required? It looks correct, but worth checking.